client/src/components/ReviewTile.js

```diff
@@ -1,13 +1,32 @@
 import React from "react";
+import { faArrowUp, faArrowDown } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const ReviewTile = ({ review, handleVote }) => {
+
+  const onClickHandler = (event) => {
+    handleVote(event.currentTarget.id, review.id)
+  }
 
-const ReviewTile = ({ review }) => {
-  
   return (
-    <div className="callout">
-      <h5>User: {review.username}</h5>
-      <h5>Created At: {new Date(review.createdAt).toString()}</h5>
-      <h5>Score: {review.score}</h5>
-      <h5>Review: {review.body}</h5>
+    <div className="callout review-tile">
+      <div className="column small-10">
+        <h5>User:{review.username}</h5>
+        <h5>Created At: {review.createdAt}</h5>
+        <h5>Score: {review.score}</h5>
+        <h5>Review: {review.body}</h5>
+      </div>
+      <ul className="vote column">
```

Comment:
ideally the vote buttons are only available to users who are signed in, otherwise anyone can vote